TFLiteSwift-Vision/Classes/TFLiteVisionInterpreter.swift

```diff
@@ -374,9 +374,18 @@ extension TFLiteVisionInterpreter {
 }
 
 extension TFLiteVisionInterpreter {
-    public enum Accelerator {
+    public enum Accelerator: Int, CaseIterable {
```

Comment:
What was the purpose of this change? When do we need this?

Reply:
@Seonghun23 
Use it int posenet demo[1][2]. it's absurd a little bit when I think again now..  hmm..

[1] https://github.com/tucan9389/TFLiteSwift-Vision/pull/23/files#diff-c15da3fd760f846eba7189f7c8b20096312ef70639e8255580603a93587333c2R93
[2] https://github.com/tucan9389/TFLiteSwift-Vision/pull/23/files#diff-c15da3fd760f846eba7189f7c8b20096312ef70639e8255580603a93587333c2L95

Reply:
I guess it helps to set UI.
In my opinion, we don't need to think about UI in the library. Because we couldn't know that. If they need it for their project, it can be implemented easily on the project.

Reply:
Sure, but I think CaseIterateble is useful even if it was not in UI context like test code. 
How about that we change to make all defined enums CaseIteratable?

Reply:
I agree it is useful in test code. It can be useful to define all enums as CaseIteratable.

Reply:
@Seonghun23 
Thanks for your opinion. I'll check all enum and inherit `CaseIteratble`. But actually, I'm not sure that changing the code in the library for test code is reasonable. I'll think about it more simultaneously.